internal/orchestrator/updates.go

```diff
@@ -616,7 +616,7 @@ func (o *Orchestrator) queueFollowingCondition(ctx context.Context, cond *rctype
 	if active != nil && active.State == rctypes.Pending {
 		byt := active.MustBytes()
 		subject := fmt.Sprintf("%s.servers.%s", o.facility, active.Kind)
-		err := o.streamBroker.Publish(ctx, subject, byt)
+		err := o.streamBroker.Publish(ctx, subject, byt, false)
```

Comment:
We should change the stream broker to separate Publish into 2 APIs, `Publish` and `PublishOverwrite`, each a public wrapper around a private `_publish` that takes the boolean. Otherwise you're making it harder to understand for code maintenance.

Reply:
I did exactly this in rivets `v1.1.0`

Reply:
thanks for that, I'll pin on that version.